src/infra/repos/postgresql/prisma/schema.prisma

```diff
@@ -8,12 +8,13 @@ datasource db {
 }
 
 model Guardian {
-  id                      Int     @id @default(autoincrement())
+  id                      String  @id @default(uuid())
```

Comment:
Doesn't this impact the entire implementation in SignUpController? Because it reflects directly on the tests created using 'id' as type 'string'.